http/headers/content-security-policy.json

```diff
@@ -1542,10 +1542,10 @@
                 "version_added": "30"
               },
               "safari": {
-                "version_added": false
+                "version_added": "10.1"
               },
               "safari_ios": {
-                "version_added": false
+                "version_added": "10.3"
               },
```

Comment:
I think this should also be 10.1, the iOS version =/= the Safari version and it should usually be the same between iOS and macOS

Reply:
I'm not super familiar with the iOS/safari versioning, but maybe we should keep this consistent with https://github.com/justyntemme/browser-compat-data/blob/master/http/headers/upgrade-insecure-requests.json#L42-L47?

Reply:
Lets keep this consistent with the other data for now. We have open PRs that try to answer what valid Safari/ios/macos versions are, but it is difficult to get an official answer from Apple.